api/Insight_API_documentation.md

```diff
@@ -0,0 +1,1198 @@
+                    "sequence": 4294967295,
+                    "n": 0,
+                    "scriptSig": {},
+                    "coinbase": "00002f646372642f"
+                }
+            ],
+            "vout": [
+                {
+                    "value": 2.55663671,
+                    "n": 0,
+                    "scriptPubKey": {
+                        "hex": "a914f5916158e3e2c4551c1796708db8367207ed13bb87",
+                        "asm": "OP_HASH160 f5916158e3e2c4551c1796708db8367207ed13bb OP_EQUAL",
+                        "addresses": [
+                            "Dcur2mcGjmENx4DhNqDctW5wJCVyT3Qeqkx"
+                        ],
+                        "type": "scripthash"
+                    },
+                    "spentTxId": null,
+                    "spentIndex": null,
+                    "spentHeight": null
+                },
+                {
+                    "value": 0,
+                    "n": 1,
+                    "scriptPubKey": {
+                        "hex": "6a2435e201000000000000000000000000000000000000000000000000007120e314d5ed75da",
+                        "asm": "OP_RETURN 35e201000000000000000000000000000000000000000000000000007120e314d5ed75da",
+                        "type": "nulldata"
+                    },
+                    "spentTxId": null,
+                    "spentIndex": null,
+                    "spentHeight": null
+                },
+                {
+                    "value": 15.35204029,
+                    "n": 2,
+                    "scriptPubKey": {
+                        "hex": "76a914b0c9ecbd75784910a7fd183f80b1ae7f57267d6b88ac",
+                        "asm": "OP_DUP OP_HASH160 b0c9ecbd75784910a7fd183f80b1ae7f57267d6b OP_EQUALVERIFY OP_CHECKSIG",
+                        "addresses": [
+                            "Dsh5gKAtf63WuzeqxFV7vJTFkPRkE35Zaf9"
+                        ],
+                        "type": "pubkeyhash"
+                    },
+                    "spentTxId": "550d9de51d7a7b6f46d3a2f24724372c8e4c632c971ed580f29f8626836a08ff",
+                    "spentIndex": 1,
+                    "spentHeight": 123735
+                }
+            ],
+            "blockhash": "00000000000001ac29068b9ff5119f29a8530e2991aa978edc33d58b775a9aed",
+            "blockheight": 123445,
+            "confirmations": 155688,
+            "time": 1491865656,
+            "blocktime": 1491865656,
+            "valueOut": 17.908677,
+            "size": 200
+        },
+        
+        .
+        .
+        .
+        .
+
+    ]
+}
+```
+
+<br/>
+
+### /txs/ (address)
+
+**URL:**  ```GET /txs```
+
+**Description:** Retrieves all transactions by address. Transactions are returned by time in descending order.
+
+**Parameters:**
+
+| Parameter           | Type                   |  Description                   | 
+| -------------------- | ---------------------- | ---------------------- | 
+| hash              | `string`      |   Address    |  
```

Comment:
should the parameter here be `address` instead of `hash` ?

Reply:
Yes, definitely.  I missed that.

Reply:
Good catch! Copy/paste sin.